bundler.d/test.rb

```diff
@@ -3,9 +3,9 @@
   gem 'simplecov', '~> 0.9'
   gem 'spork-minitest', '0.0.3'
   gem 'single_test', '~> 0.6'
-  gem 'minitest', '~> 4.7'
-  gem 'minitest-spec-rails', '~> 4.7'
-  gem 'ci_reporter', '>= 1.6.3', '< 2.0.0', :require => false
+  gem 'minitest', '~> 5.1.0'
+  gem 'minitest-spec-rails'
```

Comment:
This should probably be pinned similarly to minitest above, in this case `~> 5.3` I think for Rails 4.1 compat.